runtime/src/tests/payout.rs

```diff
@@ -15,8 +15,10 @@
 
 //! This module tests the correct computation of rewards for validators.
 
-use crate::{payout::*, Balance, Perbill, Runtime, ACME, MILLICENTS};
+use super::*;
```

Comment:
All these tests are neat, but I was wondering if there is a way to expose some adjustable constants, so that we don't have to modify manually all the tests in future, when (and if) we are going to change inflation and reward distribution rules